keyform/views.py

```diff
@@ -31,6 +34,24 @@ class RequestView(LoginRequiredMixin, UpdateView):
     def get_success_url(self):
         return reverse('home')
 
+class RequestCommentView(LoginRequiredMixin, CreateView):
```

Comment:
This should not inherit from `CreateView` since you don't use it at all. Inherit from `View` instead.
While you're at it, you can get rid of `model`, `form_class`, and `get_success_url()`.